test/vfd_swmr_group_writer.c

```diff
@@ -310,27 +315,87 @@ main(int argc, char **argv)
     if (s.file == badhid)
         errx(EXIT_FAILURE, writer ? "H5Fcreate" : "H5Fopen");
 
-    block_signals(&oldsigs);
+    /* Writer creates two named pipes(FIFO) to coordinate two-way communication */
```

Comment:
Suggest you expand this comment to describe the communications protocol, and what happens at each step.

Reply:
I enhanced it.